src/zenml/integrations/slack/alerters/slack_alerter.py

```diff
@@ -56,9 +58,9 @@ def post(self, message: str, config: Optional[SlackAlerterConfig]) -> bool:
         Returns:
             True if operation succeeded, else False
         """
-        if not isinstance(config, BaseAlerterStepConfig):
+        if not isinstance(config, SlackAlerterConfig):
```

Comment:
This raised an error when you run the example: `RuntimeError: The config object must be of type `SlackAlerterConfig`.`.

This is because the base interface still somehow passes the `BaseAlerterStepConfig` to the step.

Reply:
Good catch. Fixed it.